src/pages/create-or-edit-question/CreateOrEditQuestion.tsx

```diff
@@ -63,16 +70,58 @@ const CreateOrEditQuestion = () => {
     })
   }
 
-  const { loading, fetchData } = useAxios({
+  const { loading, fetchData: handleCreateQuestion } = useAxios({
     service: createQuestion,
     defaultResponse: defaultResponses.object,
     fetchOnMount: false,
     onResponse,
     onResponseError
   })
 
+  const editQuestion = useCallback(
+    (params?: UpdateQuestionParams) => ResourceService.updateQuestion(params),
+    []
+  )
+
+  const { fetchData: handleEditQuestion } = useAxios<
```

Comment:
> you should destructure the loading property, and show component while loading is true, because at first render your fields are empty
do i also need to destructure loading in creating and getting question?

Reply:
Sure, in getting question by id, sorry, my mistake

Reply:
fixed